public/assets/html/file-detail-new-tab.html

```diff
@@ -0,0 +1,69 @@
+<div class="file_detail_new_tab_view Application__page _Application__page">
```

Comment:
You don't need these HTML files anymore, do you?

Reply:
No, they were starter templates before the handlebars file. Would you like me to delete them?

Reply:
Yes, please.

Reply:
Ok, removed